tests/test_cart.py

```diff
@@ -549,3 +562,168 @@ def test_total_with_discount(client, sale, request_cart, product_in_stock):
     request_cart.add(variant, 1)
     line = request_cart.lines.first()
     assert line.get_total(discounts=sales) == Price(currency="USD", net=5)
+
+
+def test_product_group():
+    group = ProductGroup([Mock(is_shipping_required=Mock(return_value=False)),
+                          Mock(is_shipping_required=Mock(return_value=False))])
+    assert not group.is_shipping_required()
+    group = ProductGroup([Mock(is_shipping_required=Mock(return_value=True)),
+                          Mock(is_shipping_required=Mock(return_value=False))])
+    assert group.is_shipping_required()
+
+
+def test_cart_queryset(customer_user):
+    saved_cart = Cart.objects.create(status=Cart.SAVED)
+    customer_cart = Cart.objects.create(user=customer_user)
+
+    anonymous_cart = Cart.objects.anonymous()
+
+    assert len(anonymous_cart) == 1
+    assert anonymous_cart[0] == saved_cart
```

Comment:
Can we change the above two to `assert anonymous_carts.filter(pk=saved_cart.pk).exists()`?